rollup.config.js

```diff
@@ -1,34 +1,52 @@
-const commonjs = require('@rollup/plugin-commonjs');
-const resolve = require('@rollup/plugin-node-resolve');
-const typescript = require('rollup-plugin-typescript');
-const pkg = require('./package.json');
-const fs = require('fs');
+const commonjs = require("@rollup/plugin-commonjs");
+const resolve = require("@rollup/plugin-node-resolve");
+const typescript = require("rollup-plugin-typescript");
+const pkg = require("./package.json");
+const fs = require("fs");
+const dedent = require("dedent");
 const globals = {
-  'whatwg-fetch': 'FakeFetch',
-  'fake-xml-http-request': 'FakeXMLHttpRequest',
-  'route-recognizer': 'RouteRecognizer',
-  'url-parse': 'urlParse'
+  "fake-xml-http-request": "FakeXMLHttpRequest",
+  "route-recognizer": "RouteRecognizer",
+  "url-parse": "urlParse"
 };
 
-const rollupTemplate = fs.readFileSync('./iife-wrapper.js').toString();
-const [banner, footer] = rollupTemplate.split('/*==ROLLUP_CONTENT==*/');
+const rollupTemplate = fs.readFileSync("./iife-wrapper.js").toString();
+const [banner, footer] = rollupTemplate.split("/*==ROLLUP_CONTENT==*/");
+
+let importableInNode = {
```

Comment:
Can you extract the importable in node part in another PR and reference https://github.com/pretenderjs/pretender/pull/269?

Reply:
Yes absolutely!